tests/ManageCourses.Tests/ImporterTests/UcasDataMigratorTests.cs

```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using Castle.Core.Logging;
 using FluentAssertions;
+using FluentAssertions.Execution;
```

Comment:
ErroneousCourseLeavesOtherCoursesAlone can we also have one for opted in provider as well

Reply:
more test coverage is being done in a follow-up PR now